src/test/resources/features/DeleteSelectionProcess.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Delete SelectionProcess
+  In order to delete a selection process instance
+  As a user with role secretary
+  I want to delete the a selection process
+
+  Background:
+    Given There is a registered administrator with username "admin" and password "password" and email "admin@sample.app"
+
+  Scenario: User with secretary role delete a selection process
+    Given I login as "admin" with password "password"
+    When I create a selection process with vacancy "sample"
+    Then I delete the selection process I have just created with vacancy "sample"
+    Then The response code is 201
```

Comment:
This code doesn't correspond to the consequences of deleting something, it is when something has been created. You should have here steps that evaluate that the expected consequences of deletion are met. For instance, `the selection process with vacancy "sample" does not exist` and check in the implementation step that if you retrieve you get a 404 or that if you findByVacancy no selection process is retrieved